models/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var config = require('../lib/config');
+
+var fs = require('fs');
+var path = require('path');
+var Sequelize = require('sequelize');
+var basename = path.basename(module.filename);
+var sequelize = new Sequelize(config('databaseURL'), {
+    logging: process.env.NODE_ENV && process.env.NODE_ENV !== 'test'
+});
+
+var db = {};
+
+fs
+    .readdirSync(__dirname)
+    .filter(function(file) {
+        return (file.indexOf('.') !== 0) && (file !== basename);
+    })
+    .forEach(function(file) {
+        if (file.slice(-3) !== '.js') return;
+        var model = sequelize['import'](path.join(__dirname, file));
+        db[model.name] = model;
+    });
+
+Object.keys(db).forEach(function(modelName) {
+    if (db[modelName].associate) {
+        db[modelName].associate(db);
+    }
+});
+
+db.sequelize = sequelize;
+db.Sequelize = Sequelize;
+
+module.exports = db;
```

Comment:
A [singleton approach worries me a little bit, since it introduces a global state in the app](https://en.wikipedia.org/wiki/Singleton_pattern). Having said that, it looks like this is the common practice with sequelize; indeed it's currently *required* practice to let us put different model definitions into different files? If that's the only reason we're using it, I'd prefer to just put everything into a big `models.js` file instead.